src/Entities/Address.java

```diff
@@ -0,0 +1,63 @@
+package Entities;
+
+public class Address {
+    private int id;
+    private String street;
+    private String city;
+    private String country;
+    private String addressLine;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getStreet() {
+        return street;
+    }
+
+    public void setStreet(String street) {
+        this.street = street;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public String getCountry() {
+        return country;
+    }
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    public String getAddressLine() {
+        return addressLine;
+    }
+
+    public void setAddressLine(String addressLine) {
+        this.addressLine = addressLine;
+    }
+
+    public Address() {}
```

Comment:
constructorii trebuie sa vina dupa campuri, inainte de getteri si setteri, vina e a mea din cum am pus in cerinte, nu m-am exprimat corect